cmds/mermaid.h

```diff
@@ -0,0 +1,256 @@
+const char *INTERACTIVE_OPTION_HTML = R""(
```

Comment:
I think this header is not needed. Instead, the contents of this variable can be in a separate file such as `mermaid.html` and it can be written as follows and `cmds/dump.c` can directly use it.

In `cmds/dump.c`,
```c
const char *interactive_option_html =
#include "utils/mermaid.html"
;
```
In `utils/mermaid.html`,
```html
R""(
<style>
#options {
  border-collapse: collapse;
  width: 100%;
}
    ... (put all the remaining contents html related) ...
</table>
<br>
</details>
)""
```